docs/spec/auth/jwt.md

```diff
@@ -68,7 +68,9 @@ Token has 3 main parts:
     signing algorithm used to produce the signature. It also must have a "kid"
     field, representing the ID of the key which was used to sign the token.
 
-    The "kid" field has to be in a libtrust fingerprint compatible format.
+    The "kid" field has to be in a [libtrust](https://github.com/docker/libtrust)
```

Comment:
How many auth server implementation actually use this kid format? Can we deprecate the libtrust kid format?

Reply:
To be honest, I am not sure, so don't wanna make any false statements 😬

Reply:
Does go-jose handle the kid at all, or does it leave it up to the consumer?

Reply:
It does it for the `JSONWebKey` indeed https://pkg.go.dev/github.com/go-jose/go-jose/v3#JSONWebKey, but note that we also need to generate it from the provided trusted keys (see the constructor of the `accessController` and the token verification checks, etc.)

Reply:
I was really asking, if we didn't use the libtrust format, would JOSE provide anything useful.

Personally, I think removing the libtrust format makes sense.